packages/sdk/src/restclient.spec.ts

```diff
@@ -179,6 +186,73 @@ describe("RestClient", () => {
     });
   });
 
+  describe("blocksLatest", () => {
+    it("works", async () => {
```

Comment:
Nice test coverage

Any idea the meaning of those version fields?
I guess we could check them (at least block==10) in the RestClient and then raise an error/warning if the block height is different (meaning different header format)